src/panels/config/helpers/forms/ha-input_number-form.ts

```diff
@@ -181,15 +181,18 @@ class HaInputNumberForm extends LitElement {
     const target = ev.target as any;
     const configValue = target.configValue;
     const value =
-      target.type === "number" ? Number(ev.detail.value) : ev.detail.value;
+      target.type === "number"
+        ? Number(target.value)
+        : ev.detail.value || target.value;
```

Comment:
`detail` is undefined on an input event I think?

Reply:
Its for the icon picker

Reply:
yes, but will throw of other inputs

Reply:
Oh I see what you are saying.